src/plugins/intel_gpu/include/intel_gpu/plugin/graph.hpp

```diff
@@ -53,6 +53,10 @@ class Graph {
     std::shared_ptr<cldnn::engine> GetEngine() const { return getContextImpl(m_context)->GetEngine(); }
     int GetMaxDynamicBatchSize() const { return getConfig().max_dynamic_batch; }
     const std::map<std::string, cldnn::layout>& GetInputLayouts() const { return m_program->GetInputLayouts(); }
+    const InferenceEngine::InputsDataMap GetNetworkInputs() const { return m_program->GetNetworkInputs(); }
```

Comment:
[random spot] Please fix the logic of QueryNetwork to allow dynamic batch case (for now all nodes with dyn shapes are marked as unsupported)

Reply:
Fixed